src/util/e2eClickElement.ts

```diff
@@ -0,0 +1,33 @@
+import { E2EElement, E2EPage } from '@stencil/core/testing';
+
+/* eslint-disable no-underscore-dangle */
+export async function clickElement(page: E2EPage, elements: E2EElement[]) {
```

Comment:
What is `elements` here? Given the name of the function I would expect just a single element, what are the other elements used for? :slightly_smiling_face: :thinking:

Reply:
Yeah, I think I need to add a docblock here…

If you just send in a single element, it clicks that. But if your element is inside a shadowDOM, you need to give it the respective shadowRoot hosts too…

I'll see what I can come up with to make it clearer 👍

Reply:
Ah… I think this was a completely redundant invention by me… 🤦‍♂️

![image](https://user-images.githubusercontent.com/154725/142847504-abc38716-cf79-4104-81d0-24628f6fb5db.png)